src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -3173,7 +3308,14 @@ def _get_all_fluids(self) -> List[str]:
         """
 
         # Update this tuple when a new PVT Model is added.
-        known_pvt_models = ("compositional", "combined", "correlations", "tables")
+        known_pvt_models = (
+            "compositional",
+            "combined",
+            "correlations",
+            "pt_table_parameters",
+            "ph_table_parameters",
+            "tables",
```

Comment:
Shouldn't `tables` be replaced by two types, like: `pt_tables` and `ph_tables`?
Why did you add the `xx_table_parameters` instead of replacing the original `tables`?

Reply:
The `tables` attribute is a placeholder (dictionary) to reveice the model name and the `Path` of the table. That way we could use the `tables` attribute to specify a `pt` or `ph` description, for example.